CITATION.cff

```diff
@@ -0,0 +1,43 @@
+cff-version: 1.2.0
+message: "Cite this paper whenever you use Julia"
+authors:
+- family-names: "Bezanson"
+  given-names: "Jeff"
+- family-names: "Edelman"
+  given-names: "Alan"
+- family-names: "Karpinski"
+  given-names: "Stefan"
+- family-names: "Shah"
+  name-particle: "B"
+  given-names: "Viral"
+title: "Julia: A fresh approach to numerical computing"
+version: "v1"
+license: "MIT"
+doi: "10.1137/141000671"
+date-released: 2017-02-07
+url: "https://github.com/JuliaLang/julia"
```

Comment:
Which of the following makes the most sense for `url`?
1. `https://github.com/JuliaLang/julia`
2. `https://julialang.org`
3. `https://epubs.siam.org/doi/10.1137/141000671`
3. `https://dx.doi.org/10.1137/141000671` (which redirects to number 3 above)

Reply:
Good point. I think we should use julialang.org. This is referring to Julia the system, not Julia the codebase.

Reply:
```suggestion
url: "https://julialang.org"
```